### ISSUE TITLE
Enable view of previous versions of competencies and frameworks, and rollback.

### ISSUE THREAD

**Issue by Lomilar** [cassproject/cass-editor#499]
Will require ways to explore versioning. See referenced issues.

**Comment by stuartasutton** [comment]
As far as I am concerned, this relates to https://github.com/CredentialEngine/vocabularies/issues/563 on ghosting and that any solution should be navigable (i.e., linked data).

**Comment by Lomilar** [comment]
I agree, though I do question if history tracking should be at the level of the data schema instead of at the data format or system / content-negotiation / protocol level. 

Taking into account what you said about the value of describing what things were vs now, this probably warrants a substantial discussion. 

Git, for instance, has methods of describing what things were and how they have changed in great detail, and it is schema and format neutral, It'd need some sort of content negotiation layer to be HTTP compliant, but illustratively, that makes some sense.

**Comment by stuartasutton** [comment]
This needs a synchronous discussion with Jeanne present.

**Comment by Lomilar** [comment]
Method: The owner of an object should be able to retrieve its history (the part of the history that the owner was an owner).

Affects: https://github.com/Eduworks/ec/issues/35

**Comment by siuc-nate** [comment]
Can this be closed?

**Comment by woodkri** [comment]
We haven't started work on this yet.
